vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -166,6 +165,17 @@ function Install-ContainerD {
     $newPaths = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$installDir"
     [Environment]::SetEnvironmentVariable("Path", $newPaths, [EnvironmentVariableTarget]::Machine)
     $env:Path += ";$installDir"
+
+    $containerdConfigPath = [Io.Path]::Combine($installDir, "config.toml")
+    # enabling discard_unpacked_layers allows GC to remove layers from the content store after
+    # successfully unpacking these layers to the snapshotter to reduce the disk space caching Windows containerd images
+    (containerd config default)  | %{$_ -replace "discard_unpacked_layers = false", "discard_unpacked_layers = true"}  | Out-File  -FilePath $containerdConfigPath -Encoding ascii
```

Comment:
If no configuration file is provided, the default configurations will be used. `discard_unpacked_layers` is unique across the setting and is the only difference between the default configuration and the currently used one.
The configuration file will be overwritten during provisioning the windows node and installing containerd